common/lib/xmodule/xmodule/modulestore/django.py

```diff
@@ -273,7 +273,7 @@ def __init__(self, block=None):
         its own dedicated translation catalog along with its implementation.
         """
         self.translator = django.utils.translation
-        if block:
+        if block and getattr(block, 'unmixed_class', None):
             xblock_resource = block.unmixed_class.__module__
```

Comment:
Rather than checking for the existance of `unmixed_class`, you could just use `getattr(block, 'unmixed_class', block.__class__)`. That way, you'll get the class of the block whether or not it has a mixin attached.